config/config.yaml

```diff
@@ -85,6 +85,8 @@ telemetry:
      - "(\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+Z).+"      # 2023-09-15T11:20:36.123425532Z log
     oc_cli_path: /usr/bin/oc                                # optional, if not specified will be search in $PATH
     events_backup: True                                     # enables/disables cluster events collection
+    elastic_url: ""                                         # To track results in elasticsearch, give url to server here
```

Comment:
Tested and it works as expected! Thoughts on documenting info on the document/json that gets pushed when elastic vars are set?